test/integration/targets/aws_ec2_inventory/playbooks/create_inventory_config_with_constructed.yml

```diff
@@ -0,0 +1,32 @@
+---
+- hosts: 127.0.0.1
+  connection: local
+  gather_facts: no
+  tasks:
+    - name: write inventory config file
+      copy:
+        dest: ../test.aws_ec2.yml
+        content: |
+          plugin: aws_ec2
+          aws_access_key_id: '{{ aws_access_key }}'
+          aws_secret_access_key: '{{ aws_secret_key }}'
+          aws_security_token: '{{ security_token }}'
+          regions:
+            - '{{ aws_region }}'
+          filters:
+            tag:Name:
+              - '{{ resource_prefix }}'
+          keyed_groups:
+            - key: 'security_groups|json_query("[].group_id")'
+              prefix: 'security_groups'
+            - key: 'tags'
+              prefix: 'tag'
+            # why no work
```

Comment:
Hm. Maybe it would be good to include a "virtual key" or something that would pull out the region automatically and put it under `placement` so users could have a `placement.region` without needing to know quite as much Python.